app/src/main/res/values/strings.xml

```diff
@@ -812,4 +812,15 @@
         you follow.\n\nTo explore accounts you can either discover them in one of the other timelines.
         For example the local timeline of your instance [iconics gmd_group]. Or you can search them
         by name [iconics gmd_search]; for example search for Tusky to find our Mastodon account.</string>
+    <string name="help_empty_conversations">Here are your <b>private messages</b>; sometimes called conversations or direct messages (DM).
+        \n\nPrivate messages are created by setting the visibility [iconics gmd_public] of a post to [iconics gmd_mail] \"Direct\" and
+        mentioning one or more users in the text.
+        \n\nFor example you can start on the profile view of an account and tap the create button [iconics gmd_edit] and change the visibility.
+    </string>
+    <string name="help_empty_lists">This is your <b>lists view</b>. You can define a number of private lists and add accounts to that.
+        \n\n
+        NOTE that you can only add accounts you follow to your lists.
+        \n\n
+        These lists can be used as a tab in Account preferences [iconics gmd_account_circle] [iconics gmd_navigate_next] Tabs.
```

Comment:
gmd_account_circle is not the correct one here.

It should be manage_account. But that is not present in GoogleMaterial.kt.

Reply:
The respective file is 3 years old: https://github.com/mikepenz/Android-Iconics/blob/develop/google-material-typeface/google-material-typeface-library/src/main/java/com/mikepenz/iconics/typeface/library/googlematerial/GoogleMaterial.kt